docs/UserGuide.md

```diff
@@ -19,11 +19,11 @@ So say goodbye to traditional address book applications and say hello to the fut
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/AY2324S2-CS2103T-T12-2/tp/releases/tag/v1.3(final)).
+1. Download the latest `LookMeUp.jar` from [here](https://github.com/AY2324S2-CS2103T-T12-2/tp/releases/tag/v1.3(final)).
 
 1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
 
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar addressbook.jar` command to run the application.<br>
+1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar LookMeUp.jar` command to run the application.<br>
```

Comment:
I think it'll be better to change to the format (as shown in picture below) that we'll submit for v1.4:
https://nus-cs2103-ay2324s2.github.io/website/schedule/week13/project.html
![image](https://github.com/AY2324S2-CS2103T-T12-2/tp/assets/65162027/177f3f72-66e3-444c-b31a-0a8785a61e95)